portal/src/main/java/com/liferay/faces/portal/render/internal/PortalTagRendererCompat.java

```diff
@@ -28,8 +28,12 @@
  */
 public abstract class PortalTagRendererCompat extends Renderer {
 
+	// Public Constants
+	public static final String AUI_FORM_USE_NAMESPACE = "aui:form:useNamespace";
+	public static final String AUI_FORM_PORTLET_NAMESPACE = "aui:form:portletNamespace";
```

Comment:
@ngriffin7a not at all, I've put there just in case other Liferay versions change that string (compat class), but it may be set in other place as private.
Additionally, if we change to other class, portal-2.x changes for this LPS would be different.